recipes/reprozip/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "reprozip" %}
+{% set version = "1.0.9" %}
+{% set sha256 = "20412ac156fb8cdbc62a26890f5dd92d403c923755d84d739e1316cac252dd2b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [not linux32 and not linux64]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - sqlite
```

Comment:
I don't know whose "fault" it is really, but conda-build doesn't set these variable normally.  Rather, it sets the PREFIX and some other variables and then lets the recipe pass them on to the build system.  This package, because it is based on setuptools, probably isn't configured to take a prefix by default.  You might want to try adding `--prefix="${PREFIX}"` to the setup.py install line.  But if you have something working, that is what is important :)